### PR TITLE
Change info endpoint to be secure and unexposed by default

### PR DESCRIPTION
**Pull request by hatefpalizgar**
This is a PR related to issue #24533

### CODE REVIEW

File: spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/AccessLevel.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
 	/**
 	 * Restricted access to a limited set of endpoints.
 	 */
-	RESTRICTED("", "health", "info"),
+	RESTRICTED("", "health"),
```
Comment (mbhave) [inline spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/cloudfoundry/AccessLevel.java:34]: We do not need to alter the behavior of the CloudFoundry actuator endpoints as they are secured via a different mechanism.

**Comment by mbhave** [comment]
Thanks for the PR @hatefpalizgar. I have left a comment regarding the CloudFoundry actuators. Would you be able to update the PR accordingly?

**Comment by hatefpalizgar** [comment]
@mbhave Thanks for the comment. it is applied now.

File: spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-secure-webflux/src/test/java/smoketest/secure/webflux/SampleSecureWebFluxCustomSecurityTests.java [hunk 2]
```diff
@@ -55,10 +55,9 @@ void userDefinedMappingsSecure() {
 	}
 
 	@Test
-	void healthAndInfoDoNotRequireAuthentication() {
+	void healthDoNotRequireAuthentication() {
```
Comment (PascalSchumacher) [inline spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-secure-webflux/src/test/java/smoketest/secure/webflux/SampleSecureWebFluxCustomSecurityTests.java:58]: I guess this should be `healthDoesNotRequireAuthentication`?

Comment (mbhave) [reply spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-secure-webflux/src/test/java/smoketest/secure/webflux/SampleSecureWebFluxCustomSecurityTests.java:58]: yes

Comment (hatefpalizgar) [reply spring-boot-tests/spring-boot-smoke-tests/spring-boot-smoke-test-secure-webflux/src/test/java/smoketest/secure/webflux/SampleSecureWebFluxCustomSecurityTests.java]: fixed

**Comment by mbhave** [comment]
Thanks for making your first contribution to Spring Boot, @hatefpalizgar. This has now been merged into master along with [this polish commit](https://github.com/spring-projects/spring-boot/commit/d07e351eff1ea0dc083fea8f705f2cfd9e0f89dc).

**Comment by hatefpalizgar** [comment]
> Thanks for making your first contribution to Spring Boot, @hatefpalizgar. This has now been merged into master along with [this polish commit](https://github.com/spring-projects/spring-boot/commit/d07e351eff1ea0dc083fea8f705f2cfd9e0f89dc).

Thank you @mbhave and this is my pleasure to join this fantastic community

**Pull request closed without merging**